packages/web/src/common/store/cache/collections/sagas.js

```diff
@@ -77,8 +77,10 @@ function* createPlaylistAsync(action) {
   const userId = yield select(getUserId)
   const uid = action.playlistId
   const getFeatureEnabled = yield getContext('getFeatureEnabled')
-  const playlistEntityManagerIsEnabled =
-    getFeatureEnabled(FeatureFlags.PLAYLIST_ENTITY_MANAGER_ENABLED) ?? false
```

Comment:
Previously we weren't awaiting `getFeatureEnabled` so this was just returning a Promise which is truthy. So `playlistEntityManagerIsEnabled` was always true in this file :O

Reply:
omg